docs/developer/extending/apps/developing-apps/self-hosting-saleor-apps.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Self-hosting Saleor Apps
+---
+
+# Self-hosting Saleor Apps
```

Comment:
some of the apps may have more advanced setup. I think its good to have an introduction article but eventually we need to redirect to the specific apps for details. Eg some apps will require dynamodb

Reply:
That was an existing guide that I moved to separate section, so that we have seperate examples and guide how to run app store apps locally